admin/nodes/templatetags/node_extras.py

```diff
@@ -0,0 +1,9 @@
+from django import template
+from django.core.urlresolvers import reverse
+
+register = template.Library()
+
+
+@register.filter
+def reverse_node(value):
+    return reverse('nodes:node') + '?guid={}'.format(value)
```

Comment:
You could do ` return '{}?guid={}'.format(reverse('nodes:node'), value)` instead of using the dreaded plus sign.